request-management-api/requirements.txt

```diff
@@ -59,4 +60,5 @@ tornado==6.1
 urllib3==1.26.4
 flask-expects-json==1.5.0
 redis==3.5.3
+flask_jwt_oidc==0.3.0
```

Comment:
Small side note.  I saw the flask_jwt_oidc use around ~30% of CPU on other big projects that would be decoding JSON tokens for every HTTP request.  I think it's a solid library and unavoidable to use, but just be aware.